Source/FieldSolver/FiniteDifferenceSolver/FiniteDifferenceSolver.H

```diff
@@ -210,7 +216,7 @@ class FiniteDifferenceSolver
             std::array<std::unique_ptr<amrex::MultiFab>,3> const& Bfield,
             amrex::Real const dt);
 
-        void EvolveRhoCartesianECT (
+        static void EvolveRhoCartesianECT (
```

Comment:
Does this need to be static?
```suggestion
        void EvolveRhoCartesianECT (
```

Looks like you call this from inside the same class.

Reply:
It doesn't actually need to be static but naively I thought a function should be static every time it can be static. Is it not the case? I can make it not static otherwise